helm/app-operator/values.yaml

```diff
@@ -38,6 +38,18 @@ project:
   commit: "[[ .SHA ]]"
 
 deployment:
-  requests:
-    cpu: 250m
-    memory: 250Mi
+  management:
+    requests:
+      cpu: 250m
+      memory: 250Mi
+    limits:
+      memory: 280Mi
+  workload:
+    requests:
+      cpu: 100m
+      memory: 50Mi
+    limits:
+      memory: 50Mi
```

Comment:
The resources will be managed by VPA but it uses the ratio between requests and limits.
https://medium.com/infrastructure-adventures/vertical-pod-autoscaler-deep-dive-limitations-and-real-world-examples-9195f8422724

These are set based on my testing and with this we put back the memory limit.